src/app/subscribe/page.jsx

```diff
@@ -0,0 +1,268 @@
+"use client";
+import React, { useState } from 'react';
+import { Box, Typography, useTheme } from '@mui/material';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
+import { styled } from '@mui/system';
+import { packages } from './data';
+import { ServiceHeading } from "../../components/Home/ServicesOverview/ServiceColumnGroup";
+import {DecorativeBackgroundImage} from "../../components/Decorative/ItemBoxes";
+import RadialCircle from "../../components/Decorative/RadialCircle";
+
+const colors = ["#5DFA48", "#005BAC", "#BA8B1D"];
+const gradients = [
+    { top: '#5DFA48', bottom: '#38E274' },
+    { top: '#40A7FF', bottom: '#1C79CC' },
+    { top: '#F2DB01', bottom: '#D6AB01' },
+];
+
+const StyledCard = styled(Box)(({ theme }) => ({
+    width: '393px',
+    height: '795px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    borderRadius: '15px',
+    boxShadow: '0px 4px 30.1px rgba(0, 0, 0, 0.25)',
+    backgroundColor: theme.palette.mode === 'dark' ? "rgba(255,255,255,0.05)" : '#fff',
+    border: theme.palette.mode === 'dark' ? '1px solid rgba(255, 255, 255, 0.2)' : 'none',
+    overflow: 'hidden',
+    position: 'relative',
+    transition: 'all 0.3s ease',
+
+    "@media (max-width: 430px)":  {
+        width: "300px",
+    }
+}));
+
+const StyledImageContainer = styled(Box)(({ highlightColor, theme }) => ({
+    width: '100%',
+    height: '260px',
+    overflow: 'hidden',
+    backgroundColor: highlightColor,
+    clipPath: "path('M -2 265 Q 200 160 393 200 L 393 -2 L -2 -2 Z')",
+    boxShadow: theme.palette.mode === 'light' ? '0px 10px 30.1px rgba(0, 0, 0, 1)' : 'none',
+    '&::before': {
+        content: '""',
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+        backgroundColor: highlightColor,
```

Comment:
Move all these components to a `ComponentName.style.js` file, add the use client directive there and import necessary dependencies back into this file